src/helpers/transformErrorHandler.js

```diff
@@ -0,0 +1,16 @@
+import ResponseService from '../services/response.service';
+
+const transformErrorHandler = (schema, body, res, next) => {
+  const { error } = schema.validate(body);
+
+  if (error) {
+    const errors = [];
+    const { details } = error;
+    details.map(({ message }) => errors.push(message.split('"').join(' ')));
```

Comment:
.map returns a new array. Assign a variable to the function instead of pushing to a new array.